app/view/Warrior.svelte

```diff
@@ -0,0 +1,35 @@
+<script>
+  import { useScale } from '../context';
+
+  export let warrior;
+  export let x;
+  export let y;
+
+  const scale = useScale();
+  const size = 8;
+</script>
+
+{#if warrior}
+  <svg
+    class='warrior'
+    style={`
+      left: ${x}px;
+      top: ${y}px;
+      transform: translate(-50%, -50%) translateY(-${size * $scale}px) scale(${$scale});
+    `}>
+    <use xlink:href={`../image/piece/warriors/token.${warrior.faction}-warrior.svg#warrior`} />
```

Comment:
Do this work after the assets are bundled? Dynamic references to assets don't usually work, that's why the other images all get imported and set by function.

Reply:
Damn, in current view no, I made a typo. But rollup is configured to keep all images structure after bundling, so if just replace '../' with './' it will work

What do you think about changing all image paths on dynamic? It could make code easier to read. I can do it next thing

Reply:
Oh cool, did not realize Rollup was configured for that... where will the resulting paths be relative to? The `<base>` tag in the HTML, or the source file path? If relative to the `<base>` tag you can change those big import files to just be a function that outputs the dynamic path yes! Thanks

Reply:
All assets are just copied to `dist` folder, so I think it will work fine with `<base>`. But there may be warnings in IDE, because paths only resolving after processing (both dev and prod)

Okay, I will take care of it :)

Btw, how do you test compiling? If just build app and run html file in browser it will not be able to connect to game server

Reply:
What do you mean test compiling? I always have the server running too, and then I just play the game by myself to test everything

Reply:
Oh, okay than. I meant `rollup -c`, just build without serving. Because of you first question about asset bundling I thought that you also test this way